go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/ethereum/go-ethereum
 
-go 1.19
+go 1.22
+
+toolchain go1.22.1
```

Comment:
This was forced by the Go compiler since we updated to the lastest `go-verkle`, see its [go.mod](https://github.com/ethereum/go-verkle/blob/master/go.mod#L3).